packages/web-runtime/src/store/index.ts

```diff
@@ -4,16 +4,12 @@ import auth from './auth'
 import config from './config'
 import user from './user'
 import navigation from './navigation'
-import spaces from './spaces'
-
-const strict = process.env.NODE_ENV === 'development'
```

Comment:
This causes issues because spaces are now sitting in pinia store (spaces store) as well as in vuex store (files store). Since this only affects dev environments and we will get rid of vuex eventually, I'd propose to disable strict mode.